### PR TITLE
Added default value to 0 for ERC20 transfer

### PR DESCRIPTION
**Pull request by jpalvarezl**

### CODE REVIEW

File: src/models/backend/transfers.rs [hunk 1]
```diff
@@ -39,13 +40,29 @@ pub struct Erc20Transfer {
     pub block_number: u64,
     pub transaction_hash: String,
     pub to: String,
+    #[serde(deserialize_with = "default_if_null")]
```
Comment (rmeissner) [inline src/models/backend/transfers.rs:43]: Should we do the same for tokenId in an erc721 transfer?

File: src/models/backend/transfers.rs [hunk 2]
```diff
@@ -39,13 +40,29 @@ pub struct Erc20Transfer {
...
     pub token_address: String,
     #[derivative(Hash = "ignore")]
     pub token_info: Option<TokenInfo>,
     pub from: String,
 }
 
+#[derive(Derivative, Deserialize, Debug, Clone)]
+#[derivative(Hash)]
+#[serde(rename_all = "camelCase")]
+pub struct Erc20TransferMalformed {
```
Comment (rmeissner) [inline src/models/backend/transfers.rs:54]: Not a big fan of this :/

Comment (jpalvarezl) [reply src/models/backend/transfers.rs:54]: Oh wait, that's totally unnecessary actually, just forgot to clean up.

File: src/models/backend/transfers.rs (hunk 1 above)
Comment (jpalvarezl) [reply src/models/backend/transfers.rs:43]: I am not sure if `tokenId = 0` has a special meaning. I checked the standard quickly and seems to be ok. If you can think of a situation where `tokenId = 0` may be misunderstood let me know.

Comment (rmeissner) [reply src/models/backend/transfers.rs:43]: it would be similarly confusing to a user as a wrong value amount, imo

**Comment by jpalvarezl** [comment]
Won't fix for now

**Pull request closed without merging**